registrations/management/commands/report_pmtct_risks.py

```diff
@@ -115,7 +122,7 @@ def add_to_result(risk, reg):
                 validated=True)
 
             for registration in registrations.iterator():
-
+                add_personally_identifiable_fields(registration)
```

Comment:
This task just adds it to the current registration object, it doesn't actually save the changes to the database. So as long as we don't save it, the changes will be lost when the object gets cleaned from memory.